### PR TITLE
Try ccmake

### PR DESCRIPTION
**Pull request by scopatz**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
Can you print `CMakeFiles/CMakeError.log` after bootstrapping?

File: recipe/build.sh [hunk 1]
```diff
@@ -11,4 +14,7 @@
              -DCMAKE_BUILD_TYPE:STRING=Release \
              -DCMAKE_FIND_ROOT_PATH="${PREFIX}" \
              -DCMAKE_INSTALL_RPATH="${PREFIX}/lib"
```
Comment (isuruf) [inline recipe/build.sh]: Try adding `-DBUILD_CursesDialog=ON` here

File: recipe/build.sh [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
              --prefix="${PREFIX}" \
              --system-libs \
+             --system-libuv \
              --no-qt-gui \
              --no-system-libarchive \
              --no-system-jsoncpp \
              --parallel=${CPU_COUNT} \
              -- \
              -DCMAKE_BUILD_TYPE:STRING=Release \
-             -DCMAKE_FIND_ROOT_PATH="${PREFIX}" \
```
Comment (isuruf) [inline recipe/build.sh]: You should keep this. Directory specified here will be searched before system directories

Comment (scopatz) [reply recipe/build.sh]: good catch! I wonder how those got deleted

File: recipe/build.sh [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
              --no-qt-gui \
              --no-system-libarchive \
              --no-system-jsoncpp \
              --parallel=${CPU_COUNT} \
              -- \
              -DCMAKE_BUILD_TYPE:STRING=Release \
              -DCMAKE_FIND_ROOT_PATH="${PREFIX}" \
-             -DCMAKE_INSTALL_RPATH="${PREFIX}/lib"
+             -DCMAKE_INSTALL_RPATH="${PREFIX}/lib" \
+             -DCURSES_LIBRARY="${PREFIX}/lib/libncurses.so" \
```
Comment (isuruf) [inline recipe/build.sh]: I don't think you need this. Besides this would be `.dylib` on macosx

Comment (scopatz) [reply recipe/build.sh]: I'll try removing it/

File: recipe/build.sh [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
              --no-qt-gui \
              --no-system-libarchive \
              --no-system-jsoncpp \
              --parallel=${CPU_COUNT} \
              -- \
              -DCMAKE_BUILD_TYPE:STRING=Release \
              -DCMAKE_FIND_ROOT_PATH="${PREFIX}" \
-             -DCMAKE_INSTALL_RPATH="${PREFIX}/lib"
+             -DCMAKE_INSTALL_RPATH="${PREFIX}/lib" \
+             -DCURSES_INCLUDE_PATH="${PREFIX}/include" \
```
Comment (isuruf) [inline recipe/build.sh]: This line is unnecessary

**Pull request merged**